app/upgrades.go

```diff
@@ -33,9 +37,10 @@ var handlers = map[string]appUpgrade{
 	"neoncarrot-rc1": {}, // upgrade for 1.12.0-rc1
 	"neoncarrot":     {}, // upgrade for 1.12.0
 	"ochre-rc1": { // upgrade for 1.13.0-rc1
-		Added: []string{group.ModuleName, rewardtypes.ModuleName},
```

Comment:
Why was the group module removed here? It is a new module and will need to be registered in the upgrade...

Reply:
Thanks, that shouldn't have been removed.